examples/grid_example/public/app.tsx

```diff
@@ -50,6 +50,7 @@ import { dashboardInputToGridLayout, gridLayoutToDashboardPanelMap } from './uti
 const DASHBOARD_MARGIN_SIZE = 8;
 const DASHBOARD_GRID_HEIGHT = 20;
 const DASHBOARD_GRID_COLUMN_COUNT = 48;
+const DASHBOARD_DRAG_TOP_OFFSET = 150;
```

Comment:
This offset defines how much maximum up we want to drag the element in keyboard mode. For a natural appearance, it should be around the height of the fixed header, which is different from the grid's top position due to the content that can be before it, but not be fixed (like scrollable controls in case of dashboard). I think the best option is to keep it as a prop.